packages/app/src/app/overmind/effects/vscode/index.ts

```diff
@@ -468,14 +469,31 @@ export class VSCodeEffect {
 
     setFs(this.sandboxFsSync.create(sandbox));
 
-    if (isFirstLoad) {
-      this.sandboxFsSync.sync(() => {});
-    } else {
-      this.editorApi.extensionService.stopExtensionHost();
-      this.sandboxFsSync.sync(() => {
-        this.editorApi.extensionService.startExtensionHost();
-      });
+    return new Promise(resolve => {
+      if (isFirstLoad) {
+        // Do no start extension host
```

Comment:
I don't 100% understand this one. It says "Do not start extension host", but in the else statement it actually stops it. 🤔

Reply:
Old comment, changed to better description now 😄